### ISSUE TITLE
Feature request for date time format

### ISSUE THREAD

**Issue by johnnyktester** [2016rshah/heckle#20]
Would it be possible to support the ISO 8601 format of yyyy-mm-ddThh:mm:ssTZD as defined here https://www.w3.org/TR/NOTE-datetime for the preamble?

**Comment by 2016rshah** [comment]
Yep! You can just put in the format in this [list of supported formats](https://github.com/2016rshah/heckle/blob/master/lib/Heckle.hs#L90) and it should work :)

If you make the change, test it, and send in a pull request I would be happy to merge it!
